code/modules/reagents/reagent_containers/glass/bottle.dm

```diff
@@ -553,6 +553,13 @@ var/datum/disease2/disease/wizarditis = null
 	..()
 	reagents.add_reagent(HYPOZINE, 30)
 
+/obj/item/weapon/reagent_containers/glass/bottle/protoxin
```

Comment:
You don't need to define this, it will automatically get defined by the next line of code

Reply:
Fixed